components/editor/editor/CodeBlock.tsx

```diff
@@ -0,0 +1,52 @@
+import { NodeViewContent, NodeViewWrapper } from "@tiptap/react";
+import { ChangeEvent, FunctionComponent } from "react";
+
+// Change this for code styling
+import "highlight.js/styles/monokai-sublime.css";
```

Comment:
changes look good, check with @NiallJoeMaher re styling of the code blocks. Good job.

Reply:
I had quite a lot of difficulty getting the syntax highlighter to work in preview/render mode. Will take another look now

Reply:
https://tiptap.dev/guide/node-views#render-html

you can extend the configuration for the node in tiptap extensions - . eg Codeblock.configure{renderHTML({ HTMLAttributes }) {
  return ['my-custom-node', mergeAttributes(HTMLAttributes)]
},}

Its not essential now, we can fix in another pr

Reply:
I will try this, I went another way thats probably not ideal

Reply:
The problem is its parses as a different node. The StarterKit.Codeblock one, not the customised one, 🤔

Reply:
you should be able to configure all of that. But If what you have works, then we can add it. I don't mind to fix styles up at the end.

Reply:
technically everything can be target in global.css with .ProseMirror code, .ProseMirror p etc to make the styles sync - we can figure it out later if you wish, one source of truth should be feasable in the editors case

Reply:
I have two solutions. Maybe I should PR the alternate also, each have pros and cons I think?

Reply:
It might be worth chatting about it in the Discord too just so we can discuss too!

Reply:
Ok, will log on this evening 👍